manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: angeldev0.messyorganizer
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Angel C

```

```suggestion
Publisher: Angel Dev0
```
This should match what is written to Control Panel to ensure WinGet can detect the application once it is installed